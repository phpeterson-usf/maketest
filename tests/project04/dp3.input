-h $(repo_dir)/dp3.hex $(TESTS_DIR)/dp3.s
