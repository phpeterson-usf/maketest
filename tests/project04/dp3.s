main:
    sub r1, r2, r3
    bx lr
