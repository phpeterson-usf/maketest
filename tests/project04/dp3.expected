v2.0 raw
e0421003
e12fff1e
